.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: python
+python:
+- '3.7'
+services:
+- docker
+before_script:
+- pip install docker-compose
+- docker build --target production --tag todo-app:dev .
+- docker run -d -p 127.0.0.1:5000:5000 --env-file .env todo-app:dev
+- docker ps -a
+script:
+- docker-compose up
+- docker-compose exec -T pytest test
```

Comment:
This is trying to run the tests inside your production container even though you've added a test stage to your Dockerfile - we should be using that target to run our tests